### PR TITLE
new payment methods

### PR DESCRIPTION
**Pull request by yanikleiman**
** Checklist **
- [x] All icons have a corresponding entry in `db/payment_icons.yml`
- [x] I have followed the icon guidelines detailed in the [CONTRIBUTING.md](https://github.com/activemerchant/payment_icons/blob/master/CONTRIBUTING.md) file
- [x] I have optimized the icon with [SVGO](https://jakearchibald.github.io/svgomg/)
- [x] I am confident that all icons are clear and easy to read/understand
- [ ] I have provided a link to the brand icon’s brand guidelines whenever possible.

If this pull request is not adding new icons, you can remove this checklist.

### CODE REVIEW

**Comment by jliurner** [comment]
Team, can you please let me know if this is approved so we can start using this logos? @tauthomas01

File: app/assets/images/payment_icons/abitab.svg [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<svg version="1.1"
```
Comment (tauthomas01) [inline app/assets/images/payment_icons/abitab.svg:1]: Please read the [contributions guideline](https://github.com/activemerchant/payment_icons/blob/master/CONTRIBUTING.md)

- Remove any unused `id` attributes
- Remove any unused `class` attributes
- `viewBox` value should be `0 0 38 24`
- Run the svg code through `svgo`

**Comment by yanikleiman** [comment]
Hi @tauthomas01 I can't understand the errors of the checks that are failing. Could you help me please? Thanks!

**Comment by jliurner** [comment]
Hello @tauthomas01 , can you please provide some insights on how we can get these checks approved? Any orientation please. Thanks!

**Comment by yanikleiman** [comment]
Hi @tauthomas01 ! It seems that all tests are okey now.

File: Gemfile.lock [hunk 2]
```diff
@@ -153,4 +153,4 @@ DEPENDENCIES
   rails (>= 5.0)
 
 BUNDLED WITH
-   2.0.2
+   2.1.4
```
Comment (tauthomas01) [inline Gemfile.lock:156]: Can you revert this file? It needs to stay the same version.

File: app/assets/images/payment_icons/abitab.svg [hunk 3]
```diff
@@ -0,0 +1,6 @@
+<svg
```
Comment (tauthomas01) [inline app/assets/images/payment_icons/abitab.svg:1]: Hi, this comment applies on all other icons:

Please follow the [contributions](https://github.com/activemerchant/payment_icons/blob/master/CONTRIBUTING.md) guideline:
- The background is missing
- The border of the icon is missing
- Please run the SVG code through svgo

You can follow this [example icon](https://github.com/activemerchant/payment_icons/blob/master/app/assets/images/payment_icons/visa.svg) as a reference.

File: app/assets/images/payment_icons/abitab.svg [hunk 4]
```diff
@@ -0,0 +1,75 @@
+<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="38" height="24" role="img" viewBox="0 0 38 24" x="0px" y="0px" style="enable-background:new 0 0 38 24;" xml:space="preserve" aria-labelledby="pi-abitab">
```
Comment (tauthomas01) [inline app/assets/images/payment_icons/abitab.svg:1]: Please read the [contributions guideline](https://github.com/activemerchant/payment_icons/blob/master/CONTRIBUTING.md)

- Remove any unused `id` attributes
- Remove any unused `class` attributes
- Run the svg code through `svgo`

Comment (tauthomas01) [reply app/assets/images/payment_icons/abitab.svg:1]: Hi @yanikleiman ,

My mistake. There isn't any unused it attribute.

For the `class` attributes, we want all `<styles>` to be converted into inline attribute. What it means is the following code:

```
<style type="text/css">
  .st1{fill:#FFFFFF;}
</style>
<path xmlns="http://www.w3.org/2000/svg" class="st1" d="M35,1c1.1,0,2,0.9,2,2v18c0,1.1-0.9,2-2,2H3c-1.1,0-2-0.9-2-2V3c0-1.1,0.9-2,2-2H35"/>
```

will become

```
<path xmlns="http://www.w3.org/2000/svg" fill="FFFFFF" d="M35,1c1.1,0,2,0.9,2,2v18c0,1.1-0.9,2-2,2H3c-1.1,0-2-0.9-2-2V3c0-1.1,0.9-2,2-2H35"/>
```

File: app/assets/images/payment_icons/bancobrasil.svg [hunk 5]
```diff
@@ -0,0 +1,9 @@
+<svg
+    xmlns="http://www.w3.org/2000/svg" width="38" height="24" role="img" viewBox="0 0 38 24" aria-labelledby="pi-bancobrasil">
+    <title id="pi-bancobrasil">Banco Do Brasil</title>
+    <style>.st2{fill-rule:evenodd;clip-rule:evenodd;fill:#263675}</style>
```
Comment (tauthomas01) [inline app/assets/images/payment_icons/bancobrasil.svg:4]: This comment applies on all other icons:

For the `class` attributes, we want all `<styles>` to be converted into inline attribute. What it means is the following code:

```
<style type="text/css">
  .st1{fill:#FFFFFF;}
</style>
<path xmlns="http://www.w3.org/2000/svg" class="st1" d="M35,1c1.1,0,2,0.9,2,2v18c0,1.1-0.9,2-2,2H3c-1.1,0-2-0.9-2-2V3c0-1.1,0.9-2,2-2H35"/>
```

will become

```
<path xmlns="http://www.w3.org/2000/svg" fill="FFFFFF" d="M35,1c1.1,0,2,0.9,2,2v18c0,1.1-0.9,2-2,2H3c-1.1,0-2-0.9-2-2V3c0-1.1,0.9-2,2-2H35"/>
```

**Review by tauthomas01 — CHANGES_REQUESTED** [review]

**Comment by yanikleiman** [comment]
Hi @tauthomas01 ! Could you check if it's okey now?
Thanks!

File: app/assets/images/payment_icons/davivienda.svg [hunk 6]
```diff
@@ -0,0 +1,29 @@
+<svg
```
Comment (tauthomas01) [inline app/assets/images/payment_icons/davivienda.svg:1]: This applies to all other icons: can you increase the size of the payment brand?

[alt](https://screenshot.click/23-29-du7u7-yafwt.png)

Also can you run the svg through SVGO?

**Review by tauthomas01 — CHANGES_REQUESTED** [review]
Thank you for the changes. I left a comment that applies to all icons.

File: app/assets/images/payment_icons/davivienda.svg (hunk 6 above)
Comment (yanikleiman) [reply app/assets/images/payment_icons/davivienda.svg:1]: Hi @tauthomas01 ! Sorry for the late response. Which size should I apply? I want to finish the process of the icons but I don't undestand what I have to do. Can you help me? Thanks!!

Comment (tauthomas01) [reply app/assets/images/payment_icons/davivienda.svg:1]: Do you have an image editing tool to increase the size? Just increase it a bit so we can see the logo clearer.

Comment (yanikleiman) [reply app/assets/images/payment_icons/davivienda.svg:1]: The size should be as shown in the template, or not?

Comment (yanikleiman) [reply app/assets/images/payment_icons/davivienda.svg:1]: Hi @tauthomas01 , could you check this? Thanks!

**Comment by yanikleiman** [comment]
Hi @tauthomas01!
Could you review the PR again and see if it's everything okey?
Thanks

**Review by tauthomas01 — CHANGES_REQUESTED** [review]
Hi, this comment applies on all other icons:

- Can we increase the size of the payment brand? Because we can't see the payment brand nor the text so it's difficult to see.

**Comment by yanikleiman** [comment]
Hi @tauthomas01 
I updated the logos. Could you review again? Thanks

**Comment by larouxn** [comment]
Looks like we have some failing tests. (example failure below)

```
{:message=>"The 'pagoexpress' SVG file should have a 'viewBox' attribute on the root <svg> tag"}
```

**Comment by yanikleiman** [comment]
> Looks like we have some failing tests. (example failure below)
> 
> ```
> {:message=>"The 'pagoexpress' SVG file should have a 'viewBox' attribute on the root <svg> tag"}
> ```

Done! Coudl you check again?

**Comment by larouxn** [comment]
👋 Hey @yanikleiman, I'm sorry to inform you that we aren't accepting new icons unless they're part of the new Payments platform, which is still rolling out. You can read more on it over here: https://shopify.dev/apps/payments. 🙇‍♂️ 

https://github.com/activemerchant/payment_icons#-new-icon-contributions-are-not-being-accepted-%EF%B8%8F

**Comment by rahilvirani** [comment]
Closing the PR because it has been stale for a while. If the change is still relevant, please re-create the pull request using the new pull request template. We are sorry for the inconvenience.

**Pull request closed without merging**